ui/src/SpaceDashboard/SpaceDashboardTile.tsx

```diff
@@ -63,18 +63,35 @@ function SpaceDashboardTile({space, onClick, setCurrentModal}: SpaceDashboardTil
         return dropdownFlag;
     }
 
+    function handleKeyDownForOpenSpace(event: React.KeyboardEvent): void {
```

Comment:
These 2 functions do excatly the same thing. One with a more generic name should do the trick